### PR TITLE
Query builder

### PR DESCRIPTION
**Pull request by freva**

### CODE REVIEW

File: client/js/app/package.json [hunk 1]
```diff
@@ -35,7 +34,6 @@
     "husky": "^7",
     "lodash": "^4",
     "prettier": "2",
-    "pretty-quick": "^3",
```
Comment (ldalves) [inline client/js/app/package.json:37]: This is used by `husky` in `pre-commit`:

`npx pretty-quick --staged`

Comment (freva) [reply client/js/app/package.json:37]: Thanks, added back.

**Review by ldalves — APPROVED** [review]

**Comment by ldalves** [comment]
@freva this is really great work!!! Amazing UX with your latest changes! 😮 👏 💯 👑 🥇

**Comment by freva** [comment]
FYI: @kkraune

**Comment by kkraune** [comment]
Thanks! my local run fails on
```
Vite Error, /node_modules/.vite/deps/@mantine_notifications.js?v=b3b05ac5 optimized info should be defined (x2)
```

**Pull request merged**